recipes/modern-cpp-kafka/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
If the `cpp` wrapper requires 17, we need a `validate()` method in the recipe to check the compiler supports this standard.